.gitignore

```diff
@@ -56,3 +56,13 @@
 build/
 site/
 doxygen_output/
+
+# Ignore the object files from the Linux loader
+loader/linux/.bareflank_loader.ko.cmd
+loader/linux/.bareflank_loader.mod.o.cmd
+loader/linux/.bareflank_loader.o.cmd
```

Comment:
`loader/linux/.bareflank_loader.mod.cmd` shows up after I build as well

Reply:
Yeah, I fixed that as well.